indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -927,6 +927,7 @@ public boolean isAppendToExisting()
   {
     private static final int DEFAULT_MAX_ROWS_IN_MEMORY = 75_000;
     private static final int DEFAULT_MAX_TOTAL_ROWS = 20_000_000;
+    private static final long DEFAULT_MAX_BYTES_IN_MEMORY = Runtime.getRuntime().maxMemory() / 3;
```

Comment:
Yeah, it is being defined in all the implementation of AppenderatorConfig. What do you think, if I define a static final in this interface itself and every implementation uses that. I just tried to follow same convention as for maxRowsInMemory. The upside will be one place to change default later, downside being every implementation is tied to same default value, well,  which is same anyways even now. What do you suggest ?

Reply:
A recent patch create a `IndexTaskUtils` utility class. Maybe it would make sense to put these defaults there, like `IndexTaskUtils.DEFAULT_MAX_ROWS_IN_MEMORY` and `IndexTaskUtils.DEFAULT_MAX_BYTES_IN_MEMORY`.

Reply:
I have put these 2 defaults, but I there are others. Those can be moved to `IndexTaskUtils ` in later commits.